requirements/min-reqs.old

```diff
@@ -46,6 +46,7 @@ pydocstyle[toml]==6.1.0
 pyupgrade==2.8.0
 
 # tests
+hydra-core==1.1.0
 mypy==0.900
 nbmake==1.3.3
 omegaconf==2.1.0
```

Comment:
Have you found a way to remove the omegaconf dep?

Reply:
Just removed. It can be imported like `from hydra.compose import OmegaConf, DictConfig` instead.

Reply:
Had to upgrade from `hydra-core>=1.1.0` to `hydra-core>=1.1.1` for this